src/glwidget.cpp

```diff
@@ -2078,7 +2078,7 @@ void GLWidget::mousePressEvent(QMouseEvent* event)
         gluPerspective(60 * mScale, (GLfloat)width() / (GLfloat)height(), 0.0001, 10000.0);
         glMatrixMode(GL_MODELVIEW);
         doneCurrent();
-        mTarget = -22;
+        mTargetRoomId = -22;
```

Comment:
The original initialisation value is a bogus one!
```suggestion
        mTargetRoomId = 0;
```